parser/shared/src/main/scala/hmda/parser/fi/lar/LarParsingError.scala

```diff
@@ -0,0 +1,4 @@
+package hmda.parser.fi.lar
+
+case class LarParsingError(lineNumber: Int = 0, errorMessages: List[String] = List.empty)
+case class ParsingErrorSummary(transmittalSheetErrors: Seq[List[String]] = Seq.empty, larErrors: Seq[LarParsingError] = Seq.empty)
```

Comment:
I think that this can be a just a `transmittalSheetErrors: List[String]` and that would make it conform with the documentation added in `API.md`.